scss/switch/_layout.scss

```diff
@@ -53,7 +53,7 @@ $switch-handle-size: (.8 * $switch-size) !default;
         width: $switch-handle-size;
         height: $switch-handle-size;
         display: inline-block;
-        margin: 0 6px 0 3px;
+        margin: 0 ($padding-x-lg / 2) 0 ($padding-x-sm / 2);
```

Comment:
The left margin should be the same as the top padding of `.k-switch-container`. Perhaps, we could extract it as a new variable e.g. `$switch-padding`.

On a side note, to me it's best to drop the margin altogether and use only the padding on `.k-switch-container`.